src/main/java/com/example/examplemod/ExampleMod.java

```diff
@@ -11,10 +13,12 @@
     public static final String MODID = "examplemod";
     public static final String VERSION = "1.0";
     
+    private String bar = new String("bar"); // just do a String bar = "bar";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid instantiating String objects; this is usually unnecessary.](https://www.codacy.com/app/shardrill_t/playground/file/2177672016/issues/source?bid=3163820&fileBranchId=3163861#l16)